src/components/ColorChoice.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+
+const ColorChoice = (props) => {
+    return(
+        <section>
+            <button onClick={() => props.setColorCallback('red')}>🔴</button>
+            <button onClick={() => props.setColorCallback('orange')}>🟠</button>
+            <button onClick={() => props.setColorCallback('yellow')}>🟡</button>
+            <button onClick={() => props.setColorCallback('green')}>🟢</button>
+            <button onClick={() => props.setColorCallback('blue')}>🔵</button>
+            <button onClick={() => props.setColorCallback('purple')}>🟣</button>
+        </section>
+    );
+};
+
+//Set ColorChoice object to inherit properties/methods from ColorChoice.prototype
+ColorChoice.prototype = {
+    setColorCallback: PropTypes.func
+};
```

Comment:
I love everything about this component! The semantic HTML, the propTypes (I'm guessing L18 should be `propTypes` instead of `prototype`), the buttons!